usmqe/api/etcdapi/etcdapi.py

```diff
@@ -46,7 +46,26 @@ def get_job_attribute(self, job_id, attribute):
             job_id: id of created job
             attribute: attribute that is in given cluster
         """
-        pattern = "keys/queue/{}".format(job_id)
-        response = requests.get(pytest.config.getini("etcd_api_url") + pattern)
+        pattern = "queue/{}".format(job_id)
+        response = self.get_key_value(pattern)
+        return json.loads(response["node"]["value"])[attribute]
+
+    def get_key_value(self, key):
+        """ Get required value of a given key.
+
+        Args:
+            key: part of URI that is used in request on etcd
+        """
+
+        pattern = "keys/{}".format(key)
+        if pytest.config.getini("etcd_api_url")[4] == "s":
```

Comment:
Could you change this into something like: `pytest.config.getini("etcd_api_url").startswith("https")`? It would improve readability a lot.

Reply:
Good point, fixed.